translation/v3_to_v2.xsl

```diff
@@ -0,0 +1,157 @@
+
+XML Stylesheet Language Transformation (XSLT) to transform BuildingSync from v3 to v2
+
+Version: 0.1.0
+
+-->
+
+<xsl:stylesheet version="1.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+	xmlns:auc="http://buildingsync.net/schemas/bedes-auc/2019"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	exclude-result-prefixes="xs">
+
+  <xsl:output method="xml" version="1.0" encoding="UTF-8" omit-xml-declaration="no" indent="yes"/>
+
+  <xsl:template match="/">
+    <xsl:comment>This BuildingSync v2.3 document was generated from a BuildingSync v3.X document via an XML Stylesheet Language Transformation (XSLT).</xsl:comment>
+    <!--xsl:text>&#10;</xsl:text-->
+    <xsl:copy>
+      <xsl:apply-templates/>
+    </xsl:copy>
+  </xsl:template>
+
+  <xsl:template match="@* | node()">
+    <xsl:copy>
+      <xsl:apply-templates select="@* | node()"/>
+    </xsl:copy>
+  </xsl:template>
+
+  <!--xsl:template match="@xsi:schemaLocation">
+    <xsl:attribute name="xsi:schemaLocation">http://buildingsync.net/schemas/bedes-auc/2019 https://raw.githubusercontent.com/BuildingSync/schema/v3.0.0/BuildingSync.xsd</xsl:attribute>
+  </xsl:template-->
+
+  <xsl:template match="@version">
+    <xsl:attribute name="version">2.3.0</xsl:attribute>
+  </xsl:template>
+
+  <!-- Measures -->
+
+  <!-- 1. Map "Repair leaks in ducts" to "Repair leaks / seal ducts" -->
+  <xsl:template match="auc:MeasureName[. = 'Repair leaks in ducts']">
+    <xsl:copy>Repair leaks / seal ducts</xsl:copy>
+  </xsl:template>
+  <!-- 2. Map "Insulate attic hatch" to "Insulate attic hatch / stair box" -->
+  <xsl:template match="auc:MeasureName[. = 'Insulate attic hatch']">
+    <xsl:copy>Insulate attic hatch / stair box</xsl:copy>
+  </xsl:template>
+  <!-- 3. Map enumeration "MROE" to "MORE" under auc:eGRIDRegionCode -->
+  <xsl:template match="auc:eGRIDRegionCode/auc:MeasureName[. = 'MROE']">
+    <xsl:copy>MORE</xsl:copy>
+  </xsl:template>
+  <!-- 4. Map enumeration "Convert to cleaner fuels" to "Convert to Cleaner Fuels" under auc:BoilerPlantImprovements -->
+  <xsl:template match="auc:BoilerPlantImprovements/auc:MeasureName[. = 'Convert to cleaner fuels']">
+    <xsl:copy>Convert to Cleaner Fuels</xsl:copy>
+  </xsl:template>
+  <!-- 5. Map enumeration "Energy cost reduction through rate adjustments - uncategorized" to "Other" under auc:EnergyCostReductionThroughRateAdjustments/auc:MeasureName -->
+  <!--xsl:template match="auc:EnergyCostReductionThroughRateAdjustments/auc:MeasureName[. = 'Energy cost reduction through rate adjustments - uncategorized']">
+    <xsl:copy>Other</xsl:copy>
+  </xsl:template-->
+  <!-- 6. Map auc:FutureOtherECMs to auc:Uncategorized -->
+  <!--xsl:template match="auc:FutureOtherECMs">
+    <xsl:element name="auc:Uncategorized">
+      <xsl:apply-templates/>
+    </xsl:element>
+  </xsl:template-->
+  <!-- 7. Limit auc:MeasureName to exactly one by commenting out all other auc:MeasureNames -->
```

Comment:
@JieXiong9119 why are these commented out? Should we remove them completely?

Reply:
I kept all the v2 to v3 changes in this xslt just for references or in case some of these changes should be brought in for backward translation. We can remove them if they are not needed.

Reply:
@JieXiong9119 gotcha, could you remove them if they're not required for backward translation? and if they are could you uncomment / tweak them to work?